### PR TITLE
Add OpenGL checks to master

### PR DESCRIPTION
**Pull request by Pentarctagon**
Seeing as the dependencies have been added to master's docker images.  This would also be a couple fewer things to need to do when the opengl branch is periodically rebased against master.

### CODE REVIEW

**Comment by gfgtdf** [comment]
wait so master does already require opengl or is that a pr shodul shodul oynl be merged after an opengl branch is merged?

**Comment by Pentarctagon** [comment]
Master does not currently require it, but it's covered by just installing `libglew1.13 libglew-dev`(on [Ubuntu 16.04](https://github.com/wesnoth/wesnoth/blob/master/docker/Dockerfile-base-1604-master#L11)), and I know there's at least a couple other changes I'd like to make to these files, so adding this to master now would mean fewer files to fix merge conflicts with later.

File: SConstruct [hunk 1]
```diff
@@ -412,6 +412,14 @@ if env["prereqs"]:
         if env["history"]:
             client_env.Append(CPPDEFINES = ["HAVE_HISTORY"])
 
+        if env["PLATFORM"] == 'darwin':
+            client_env.Append(FRAMEWORKS = "OpenGL")
+            client_env.AppendUnique(LIBS = ["GLEW"])
+        elif env["PLATFORM"] == 'posix':
+            opengl_config = check_output(["pkg-config", "--libs", "--cflags", "gl", "glew"])
+            opengl_flags = client_env.ParseFlags(opengl_config)
+            client_env.MergeFlags(opengl_flags)
```
Comment (jyrkive) [inline SConstruct]: Please use `client_env.ParseConfig("pkg-config --libs --cflags gl glew")` instead. (Documented in the [SCons manual](http://scons.org/doc/production/HTML/scons-man.html).)

File: SConstruct [hunk 2]
```diff
@@ -412,14 +414,6 @@ if env["prereqs"]:
         if env["history"]:
             client_env.Append(CPPDEFINES = ["HAVE_HISTORY"])
 
-        if env["PLATFORM"] == 'darwin':
-            client_env.Append(FRAMEWORKS = "OpenGL")
-            client_env.AppendUnique(LIBS = ["GLEW"])
-        elif env["PLATFORM"] == 'posix':
-            opengl_config = check_output(["pkg-config", "--libs", "--cflags", "gl", "glew"])
-            opengl_flags = client_env.ParseFlags(opengl_config)
-            client_env.MergeFlags(opengl_flags)
```
Comment (jyrkive) [inline SConstruct]: Could you rewrite history a bit to avoid this code being added and then deleted?

Comment (Pentarctagon) [reply SConstruct]: Would you prefer I edit that commit, or squash the two commits together?

Comment (jyrkive) [reply SConstruct]: I'd prefer squashing them. I think they belong together.

**Comment by Pentarctagon** [comment]
Squashed.

**Comment by jyrkive** [comment]
Thanks. :)

**Pull request merged**